### DISCUSSION TITLE
Extension of the Westeros tutorial during the MESSAGEix Community Meeting

### DISCUSSION THREAD

**Discussion by gorkemgungormetu** [iiasa/message_ix#681] [Q&A] [answered]
I had to re-install MESSAGE-ix for some purpose and unfortunately re-installed Anaconda and lost the Jupyter Notebook files distributed during the workshop related to the extension of the Westeros tutorial.

Is it possible to have access to these files?

**Comment by behnam-zakeri** [comment]
Thanks, @gorkemgungormetu, for approaching. There were a few tutorials on calibration, representation of trade, and changing the GAMS formulation. Which one do you need?

**Reply by gorkemgungormetu** [reply]
I was planning to use them all for training in my elective course but somehow lost them during re-intallation of Conda and MESSAGE-ix. The representation of trade with region nodes was very interesting but if you could share them all that would be perfect.

**Reply by gorkemgungormetu** [reply]
Is it possible for you to share @behnam-zakeri ?

**Reply by behnam-zakeri** [reply, marked as answer]
Of course @gorkemgungormetu
- please find the tutorial related to creating a multi-node and adding trade possibilities [here](https://github.com/iiasa/message_ix/pull/683).
- you can see the tutorial for extending the GAMS formulation and adding new items to the MESSAGEix [here](https://github.com/iiasa/message_ix/pull/602).
